docs/USAGE.md

```diff
@@ -0,0 +1,16 @@
+# Instructions to run the application locally
+1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html)
```

Comment:
Mmmm, we agreed with Dmitri the supported version is LTS 11.2 @aababilov ?

Reply:
There shouldn't be a difference given the current code in the project. The current code doesn't use any features introduced in JDK9+, so you can run it on JDK8. If we introduce code that requires greater than JDK8 then we could bump this. But to my knowledge there isn't a drawback of allowing this code to run on JDK8.